plugins/generic/pln/PLNGatewayPlugin.inc.php

```diff
@@ -131,6 +155,10 @@ function fetch() {
 
                 $templateMgr->display($this->getTemplatePath() . DIRECTORY_SEPARATOR . 'ping.tpl', 'text/xml');
 
+                unset($terms);
+                unset($termsAccepted);
+                unset($termKeys);
```

Comment:
OTOH, I don't think these unsets are going to do anything, since they're right above the end of scope anyway.